cfgov/data_research/scripts/export_public_csvs.py

```diff
@@ -40,6 +43,43 @@
 logger = logging.getLogger(__name__)
 
 
+def format_size(bytecount, suffix='B'):
+    for unit in ['', 'K', 'M', 'G']:
+        if abs(bytecount) < 1024.0:
+            return "%3.1f%s%s" % (bytecount, unit, suffix)
+        bytecount /= 1024.0
+    return "%.1f%s%s" % (bytecount, 'T', suffix)
+
+
+def save_metadata(csv_size, slug, thru_date, date_value, geo_type):
+    """Save slug, URL, thru_date and file size of a new CSV download file."""
+    pub_date = "{}".format(datetime.date.today())
+    csv_url = "{}/{}.csv".format(S3_DOWNLOADS_BASE_URL, slug)
+    download_meta_file, cr = MortgageMetaData.objects.get_or_create(
+        name='download_files')
+    new_posting = {geo_type:
+                   {'slug': slug,
+                    'url': csv_url,
+                    'size': csv_size,
+                    'thru_date': thru_date,
+                    'pub_date': pub_date}}
+    if not download_meta_file.json_value:
+        download_meta_file.json_value = json.dumps(
```

Comment:
See above re: possible use of JSONField. That would eliminate the need for this kind of `json.dumps`/`json.loads` here, and be easier to migrate in future to native JSON fields if we want to do that.